thoth/adviser/steps/tensorflow_avx2.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+# thoth-adviser
+# Copyright(C) 2020 Fridolin Pokorny
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Recommend TensorFlow builds optimized for AVX2 enabled CPU processors."""
+
+from typing import Any
+from typing import Optional
+from typing import Tuple
+from typing import List
+from typing import Dict
+from typing import TYPE_CHECKING
+import logging
+
+from thoth.python import PackageVersion
+
+from ..enums import RecommendationType
+from ..state import State
+from ..step import Step
+
+
+if TYPE_CHECKING:
+    from ..pipeline_builder import PipelineBuilderContext
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class TensorFlowAVX2Step(Step):
+    """A step that recommends AICoE TensorFlow builds optimized for AVX2 enabled CPU processors."""
+
+    _SCORE_ADDITION = 0.2
+    _JUSTIFICATION_ADDITION = [
+        {
+            "type": "INFO",
+            "message": "AICoE TensorFlow builds are optimized for AVX2 instruction sets that your CPU supports",
```

Comment:
Is this message ok-ish? Maybe too personal with `your`? Do we want to have some convention on how messages should look like (besides schema used for justifications).

Reply:
How the pipeline is triggered? Checking .thoth.yaml CPU family and model I guess, right?

"AICoE TensorFlow builds are optimized for AVX2 instruction sets supported in the CPU identified for the advise. "

Maybe something like this?

Not sure about a convention on the text itself in this case. (reqs: user-friendly, minimum details to give clear) understanding)

Reply:
Yes, based on the `.thoth.yaml` configuration.

Ack, will adjust the message :+1: